gobblin-distribution/build.gradle

```diff
@@ -17,6 +17,17 @@ repositories {
   }
 }
 
+configurations {
+  compile {
+    if (project.hasProperty('excludeHadoopDeps')) {
```

Comment:
I suggest that you also check rootProject.hasProperty('excludeHadoopDeps') if it has been specified at the root level.